astroid/bases.py

```diff
@@ -5,8 +5,10 @@
 """This module contains base classes and functions for the nodes and some
 inference utils.
 """
+from __future__ import annotations
 
 import collections
+from typing import TYPE_CHECKING
```

Comment:
Not strictly necessary, but nice for syntax highlighting in VS Code.
```suggestion
import collections
import collections.abc
from typing import TYPE_CHECKING
```